common/tools/dev-tool/src/util/testProxyUtils.ts

```diff
@@ -87,37 +204,32 @@ export async function isProxyToolActive(): Promise<boolean> {
   }
 }
 
-async function makeRequest(uri: string, requestOptions: RequestOptions): Promise<IncomingMessage> {
-  return new Promise<IncomingMessage>((resolve, reject) => {
-    const req = request(uri, requestOptions, resolve);
-    req.once("error", reject);
-    req.end();
-  });
-}
+async function getTargetVersion() {
+  // Temporarily pinning to 1.0.0-dev.20221212.3 until changes made to eng/common in Jan '23.
+  return "1.0.0-dev.20221212.3";
```

Comment:
Seems fine for now. Any way we could just check the GH releases and use the latest one?

Reply:
Once the `eng/common` change is made (I think all that's actually needed is an update to the `target_version.txt` file, but I will defer to Scott) we can revert to the existing logic.

Using the latest one is not something we always want to do. There have occasionally been breaking changes to the test proxy that we need to absorb in the recorder library before we increment the version, and we use the `target_version.txt` in eng/common to control it. It's also important that the test proxy version matches what the JS recorder library expects when working on an older branch, for similar reasons.